### ISSUE TITLE
set colorscheme update colorscheme partially

### ISSUE THREAD

**Issue by Desiment** [ranger/ranger#2760]
#### Runtime Environment
- OS: Arch Linux 
- Terminal emulator: alacritty 0.11.0 (also tried with kitty and got same behavior)  
- ranger version: ranger 1.9.3
- Python version: 3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0]
- Locale: ru_RU.UTF-8

#### Current Behavior
``set colorscheme`` command does not work properly. It changes colorschme only partially. I created two simple colorschemes, for which method ``use()`` return either ``(0,0,normal)`` (zenlight) or ``(255, 255, normal)`` (zendark). After running ranger with the light colorscheme I get (as excepted):

![image](https://user-images.githubusercontent.com/36130705/207439416-97cddbcf-6f8c-4d79-afc9-cd3951a4d40d.png)

But after execution ```set colorscheme darktest``, I get:
![image](https://user-images.githubusercontent.com/36130705/207439545-4bf62c6d-ef69-4710-bbf2-f735cd1c23aa.png)

It seems that ranger applies colorscheme only in directories that were not visited before setting a new colorscheme (here I switched to some random directory):
![image](https://user-images.githubusercontent.com/36130705/207439631-af435639-bec7-4745-a8b0-59500995099f.png)

This work with default colorschemes too. Also, I tested this in kitty and noticed the same thing.

#### Expected Behavior
Running with darktest colorscheme gives 
![image](https://user-images.githubusercontent.com/36130705/207440067-674604fa-96e9-41a7-82d9-34eebf58d6f7.png)

So getting this after running ``set colorscheme zendark`` is (I think) expected behavior.

#### Steps to reproduce
1. Run ranger with some colorscheme
2. Change colorscheme

**Comment by toonn** [comment]
Yeah, this is a known bug. Thanks for the report though. Kinda has a low priority for us since changing the colorscheme while running Ranger doesn't occur that often. PRs welcome though : )
